contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -438,6 +441,22 @@ def fn(message, progress, request, **kwargs):
         yield
         self._notify = old
 
+    @contextmanager
+    def reply_filter(self, filter):
+        """Filter the fields returned from am RPC call (or more than one)..
+
+        This is a context manager and should be used like this:
+
+        ```python
+        with rpc.reply_filter({"transactions": [{"outputs": [{"amount_msat": true, "type": true}]}]}):
+            rpc.listtransactions()
+        ```
```

Comment:
This is very cool ^^

Reply:
I stole it from your context code I think!!